### DISCUSSION TITLE
Console groups

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#2373] [Ideas]
https://developer.mozilla.org/en-US/docs/Web/API/Console/group

**Comment by jonbell-lot23** [comment]
March 15 2022 update:

Another great "needle in a haystack" UX question. See also https://github.com/RecordReplay/devtools/discussions/2371

**Reply by jasonLaster** [reply]
We should differentiate between parity work to support what is in chrome devtools and features we would like to offer on top of that.

With console.groups, by default Replay should behave the same as chrome devtools. We can always add additional rollups and grouping on top
